src/services/refreshToken.js

```diff
@@ -0,0 +1,25 @@
+import authService from './authService/authService';
+import jwtDecode from 'jwt-decode';
+
+const TOKEN_REFRESH_INTERVAL = 900000;
+const DELTA = 60000;
```

Comment:
`DELTA` of what?
`TOKEN_REFRESH_INTERVAL` is used not in refresh but in logout, why is it even needed?